packages/@o3r/test-helpers/src/utilities/package-manager.ts

```diff
@@ -37,13 +43,14 @@ export function getPackageManager() {
 
 function execCmd(cmd: string, args: string, options: ExecSyncOptions) {
   try {
+    const d = Date.now();
     const output = execFileSync(
       cmd.split(' ')[0],
       [...cmd.split(' ').slice(1), ...args.split(' ')].filter((arg) => !!arg),
       {...options, shell: process.platform === 'win32', stdio: 'pipe', encoding: 'utf8'}
     );
     // eslint-disable-next-line no-console
-    console.log(`${cmd} ${args}\n${output}`);
+    console.log(`${cmd} ${args} [${Date.now() - d}ms]\n${output}`);
```

Comment:
Using performance API would be cleaner :)

Reply:
done